src/app/views/request/confirmation/confirmation.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="content">
-    <h1>I feel</h1>
+    <h1>I feel the room is</h1>
```

Comment:
The room can't be sweaty. Add the the room is to the relevant buttons "The room is windy (the "the room" part can be on the top of the button)

Reply:
Adding another parameter to the existing items is something we shouldn't do with the time we have, it would introduce a lot of new bugs.

Reply:
You're right, add "making me" to the sweaty button instead.